component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -828,6 +819,27 @@ public Response updateUserById(@Context UriInfo uriInfo,
     return EntityBuilder.getResponse(EntityBuilder.buildEntityProfile(id, uriInfo.getPath(), expand), uriInfo, RestUtils.getJsonMediaType(), Response.Status.OK);
   }
 
+  @GET
+  @Path("onBoarding/{id}")
```

Comment:
/onboard  (prefer loser case in rest endpoints)

Reply:
:+1:

Reply:
updated!